lemur/certificates/views.py

```diff
@@ -480,6 +480,9 @@ def post(self, data=None):
            :statuscode 403: unauthenticated
 
         """
+        if not service.is_valid_owner(data["owner"]):
+            return dict(message=f"Invalid owner: check if email {data['owner']} exists as a Team DL"), 412
```

Comment:
shall revise the language to group?

"Invalid owner: check if {data['owner']} is a valid group. individuals cannot be certificate owners."

Reply:
Thanks for suggestion, I'm not sure how commonly terminology "group" is used over team dl or email. So I'll do a minor tweak in `Invalid owner: check if {data['owner']} is a valid group email. Individuals cannot be certificate owners.`